packages/plugin-react-oxc/src/index.ts

```diff
@@ -141,7 +141,7 @@ export default function viteReact(opts: Options = {}): PluginOption[] {
         return [
           {
             tag: 'script',
-            attrs: { type: 'module' },
+            attrs: { type: 'module', async: 'true' },
```

Comment:
async attribute is a [boolean attribute](https://developer.mozilla.org/en-US/docs/Glossary/Boolean/HTML), so I think this should be
```suggestion
            attrs: { type: 'module', async: true },
```
or
```suggestion
            attrs: { type: 'module', async: '' },
```
or
```suggestion
            attrs: { type: 'module', async: 'async' },
```
(browsers would treat them the same though)